src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/trait/BitbucketBranchDiscoveryTrait/help.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Discovers branches on the repository.
```

Comment:
Happy to leave, but **on**  feels wrong, is it not "in" the repository?

Reply:
Happy to change these to "in". 👍